### DISCUSSION TITLE
GHOSTS in Azure

### DISCUSSION THREAD

**Discussion by cyberautomate** [cmu-sei/GHOSTS#143] [Q&A] [answered]
I pulled the docker image from here: https://hub.docker.com/r/dustinupdyke/ghosts. I pushed that image to an Azure Container Registry and now have a container instance running from the image. Should I be able to access the web interface api/home from the public IP of my container instance (EX 158.63.583.46/API/home)? Is there any additional configuration needed to get the image working with a public IP? 

I can't find anything in the docs reference this scenario.

Thanks in advance

**Comment by sei-dupdyke** [comment, marked as answer]
Hi, thanks for checking out the project - it should work in Azure just fine.

In the compose file, the ports for the API are "52388:5000", so you probably want to change that to 80 or 443 or another lower port number.

There shouldn't be anything additional that is required in the configuration to bind ghosts to a particular ip.
